docs/guide/testing-teal.md

```diff
@@ -282,7 +282,7 @@ Now, we will execute a transaction with an app call (stateful TEAL). The app is
     type: TransactionType.CallNoOpSSC,
     sign: SignType.SecretKey,
     fromAccount: john.account,
-    appId: 0,
+    appID: 0,
```

Comment:
this is API breaking, not sure if this is good at this stage.

Reply:
this needs to be done either we have to change appID to appId or vice versa, because in some params it is appID and in others it is appId, therefore we cannot group them in one

Reply:
Ah, OK.
So let's use `appID`. Please add entry to CHANGELOG

Reply:
ok